habitat_sim/utils/data/pose_extractor.py

```diff
@@ -10,7 +10,7 @@
 from habitat_sim.utils.common import quat_from_two_vectors
 
 
-class TopdownView(object):
+class TopdownView:  # noqa: SIM119
```

Comment:
This complains that it really should be a data/attrs class, which is probably true, but I don't want to refactor it now.